code/modules/power/battery_ion.dm

```diff
@@ -0,0 +1,70 @@
+// Generic battery machine
+// stores power
+
+/obj/machinery/power/battery/portable
+	name = "portable power storage unit"
+	desc = "A IOn-model portable storage unit, used to transport charge around the station."
```

Comment:
Do the ion batteries have a much lower charge? Because super magnetic energy storage would hold way more energy.